solutions/sequences/ValidParentheses.cpp

```diff
@@ -0,0 +1,55 @@
+// https://leetcode.com/problems/valid-parentheses/
+
+#include <stack> 
+
+class Solution {
+public:
+    bool isValid(string s) {
+        std::stack<char> leftParentheses;
```

Comment:
lets think about an imaginary language that has 50 types of parenthasis, how will you handle it? you will have a long "if ladder".